templates/package.xml.template

```diff
@@ -24,13 +24,9 @@
    </stability>
    <license>Apache 2.0</license>
    <notes>
-  - Fixed some memory leaks #9559, #10996
-  - Disabled cares dependency from gRPC C Core #10940
-  - De-coupled protobuf dependency #11112
-  - Fixed extension reported version #10842
-  - Added config.w32 for Windows support #8161
-  - Fixed PHP distrib test after cc files were added #11193
-  - Fixed protoc plugin comment escape bug #11025
+  - Channel are now by default persistent #11878
```

Comment:
IMO we should call attention to new features for the 1.6 release.  Anything that has already been released and noted in the 1.4 release probably don't need to be repeated here.